specs/intermediate/intermediate.go

```diff
@@ -64,11 +65,12 @@ type RepeatedParameterMap struct {
 
 // Call intermediate specification
 type Call struct {
-	Name     string        `hcl:"name,label"`
-	Endpoint string        `hcl:"endpoint,label"`
-	Type     string        `hcl:"type,optional"`
-	Request  *ParameterMap `hcl:"request,block"`
-	Rollback *RollbackCall `hcl:"rollback,block"`
+	DependsOn []string      `hcl:"depends_on,optional"`
```

Comment:
Do we able to run several calls in parallel mode if they do not have cross-dependencies?

Reply:
I got a check for cycle dependencies available inside a future PR. For now, we only need to worry about the possibility of defining dependencies.